modules/lvms-scaling-storage-expand-pvc.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// storage/logical-volume-manager-storage.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-scaling-expand-pvc_{context}"]
+= Expanding PVCs
+
+To leverage the new storage after adding additional capacity, you can expand existing persistent volume claims (PVCs) with LVM Storage.
+
+.Prerequisites
+
+* Dynamic provisioning is used.
+* The controlling `StorageClass` object has `allowVolumeExpansion` set to `true`.
+
+.Procedure
+
+. Set `.spec.resources.requests.storage` to the desired new size for the PVC.
```

Comment:
Do we need to state where to set this parameter? I assume it's in the PVC object. Will a typical user know where to make this setting?